docs/docs/guides/tutorials/userId.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Getting user Data
+sidebar_label: User data
+keywords: [user, user ID, userID, email]
+---
+
+Since every project can have a different need when handling user onboarding,
+we have put together some of the approcaches you can use to
+obtain the data needed to identify a specific user.
+
+## Get user ID from the JWT
+
+On successful authentication, Hanko retreives a JWT contained inside of a cookie. One of the the thing
+we can get from this JWT is the user ID, we can use [jose library](https://www.npmjs.com/package/jose) to decode
+the value of such JWT:
+
+```js
+import { cookies } from "next/headers";
+import * as jose from "jose";
+
+export async function userId() {
+  const token = cookies().get("hanko")?.value;
+  const payload = jose.decodeJwt(token ?? "");
+
+  const userID = payload.sub;
+  return userID;
+}
+```
+
+## Get user by ID using Hanko Admin API
+
+The [Hanko Admin API](https://docs.hanko.io/api/admin) provides detailed information about an specific user,
+using the user ID as an identifier. In the Hanko Cloud Settings, you have the option to enable the Admin API
+(it's a Pro feature) and create an `API Key`, when this last one is created the `secret` will be shown to you
+only one time, make sure you store it in a safe place.
+
+The Admin API is protected and you're gonna need to send the `secret`
+inside the `headers` in your `fetch`request:
```

Comment:
IMO the text should explain (on high level) what the code listing does e.g.: 

> The [Hanko Admin API](https://docs.hanko.io/api/admin) provides detailed information about a specific user.
To get it just call the [`/users/{id}`](https://docs.hanko.io/api/admin#tag/User-Management/operation/getUser) endpoint of the Hanko Admin API, where `id` is the user id from the JWT.

And put the API key stuff in an info section e.g.:

```
:::info

On Hanko Cloud the Hanko Admin API is a paid feature and must be enabled separately.

You also need an API key `secret` to access the Hanko Admin API. You can generate one under the `Settings/API Keys` section of your project.
Keep in mind, that the API key `secret` is only showed once, so make sure to store it in a safe place.

:::
```